src/UuidFactory.php

```diff
@@ -229,6 +231,23 @@ public function uuid5($ns, $name)
         return $this->uuidFromNsAndName($ns, $name, 5, 'sha1');
     }
 
+    public function comb()
+    {
+        $currentCodec = $this->getCodec();
+        $combGenerator = new CombGenerator($this->getRandomGenerator(), $this->getNumberConverter());
+        $combCodec = new TimestampFirstCombCodec($this->getUuidBuilder());
```

Comment:
There's a difference between a COMB UUID and a Timestamp First COMB UUID. I'm worried that that name of the method `comb()` causes confusion, since you're using the `TimestampFirstCombCodec()` internally.

Reply:
Indeed but `TimestampFirstCombCodec` is the one that produces monotonically increasing UUIDs, which is the whole point.

Do you have an alternative name in mind?

Reply:
Maybe `timestampFirstComb()`?